spec/license_checker_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'spec_helper'
+
+describe Repoman::LicenseChecker do
+  let(:repo) { double(Github::Mash, { full_name: 'dummy-repo' }) }
+  let(:checker) { described_class.new(repo) }
+  let(:check_passed) { { result: :passed } }
+  let(:license_text) { '' }
+  let(:license_url) { 'https://raw.githubusercontent.com/dummy-repo/master/LICENSE' }
```

Comment:
There's no such repo, AFAIK. That URL is never used. It's just a value that we expect to be the result of working out what URL we would use to fetch the file called 'LICENSE' from the repo called 'dummy-repo'